php/php.phpdoc/src/org/netbeans/modules/php/phpdoc/PhpDocScript.java

```diff
@@ -151,7 +151,7 @@ private String sanitizePath(String path) {
                 "run", // NOI18N
                 // params
                 "--ansi", // NOI18N
-                "--progressbar", // NOI18N
+                // "--progressbar" doesn't exist since PHPDocumentor 3
```

Comment:
I would expect everywhere the latest version of all the CLI tools we support, so even removing this line completely would be OK IMHO.

Reply:
I see. I'll do that from next time :) Thank you!

Reply:
Up to you definitely, it was just an idea.